docs/revised-stream-4-language.md

```diff
@@ -79,11 +79,11 @@ TO DO - need examples.
 
 It is important to not only provide tooling for developers to develop software with memory safety, but sharing HOW and WHY to do so is equally of value.  There are several areas where educational materials, training, and job aids should be identified or created to help developers quickly learn about memory safe languages, how to use more memory-safe coding techniques in older languages, as well as the threats and vulnerabilities non-memory-safe coding practices can incur.  We propose to create a reference library and educational materials on:
 
-- the threats and vulnerabilities that exist around memory safety coding flaws
-- the benefits of developing using memory safe languages
-- references to authoritative guides for memory-safe languages
-- educational training material for older languages, such as C & C++, on how to develop using memory-safety techniques and safe memory handling practices
-- spread awareness and help others develop skills in memory safe languages and secure coding techniques through blogs, podcasts, conference presentations, and training classes
+* the threats and vulnerabilities that exist around memory safety coding flaws
+* the benefits of developing using memory safe languages
+* references to authoritative guides for memory-safe languages
+* educational training material for older languages, such as C & C++, on how to develop using memory-safety techniques and safe memory handling practices
```

Comment:
suggestion: "C and C++", instead of "C & C++"

Reply:
ah! ty!

Reply:
Done!